kernel_hardening_checker/test_engine.py

```diff
@@ -180,16 +180,16 @@ def test_simple_cmdline(self):
         self.get_engine_result(config_checklist, result, 'json')
         self.assertEqual(
                 result,
-                [['name_1', 'cmdline', 'expected_1', 'decision_1', 'reason_1', 'OK'],
-                 ['name_2', 'cmdline', 'expected_2', 'decision_2', 'reason_2', 'FAIL: "UNexpected_2"'],
-                 ['name_3', 'cmdline', 'expected_3', 'decision_3', 'reason_3', 'FAIL: is not found'],
-                 ['name_4', 'cmdline', 'is not set', 'decision_4', 'reason_4', 'OK: is not found'],
-                 ['name_5', 'cmdline', 'is present', 'decision_5', 'reason_5', 'OK: is present'],
```

Comment:
Could you please add one extra space at each line to make it look prettier and have a smaller diff?

Reply:
Nono, I mean adding one space before each `{'option_name' for better alignment and smaller diff with the original code: 
```diff
         self.assertEqual(
                 result,
                 [{'option_name': 'CONFIG_NAME_1', 'type': 'kconfig', 'desired_val': 'expected_1', 'decision': 'decision_1', 'reason': 'reason_1', 'check_result_text': 'OK', 'check_result': True},
-                {'option_name': 'CONFIG_NAME_2', 'type': 'kconfig', 'desired_val': 'expected_2', 'decision': 'decision_2', 'reason': 'reason_2', 'check_result_text': 'FAIL: "UNexpected_2"', 'check_result': False},
-                {'option_name': 'CONFIG_NAME_3', 'type': 'kconfig', 'desired_val': 'expected_3', 'decision': 'decision_3', 'reason': 'reason_3', 'check_result_text': 'FAIL: is not found', 'check_result': False},
+                 {'option_name': 'CONFIG_NAME_2', 'type': 'kconfig', 'desired_val': 'expected_2', 'decision': 'decision_2', 'reason': 'reason_2', 'check_result_text': 'FAIL: "UNexpected_2"', 'check_result': False},
+                 {'option_name': 'CONFIG_NAME_3', 'type': 'kconfig', 'desired_val': 'expected_3', 'decision': 'decision_3', 'reason': 'reason_3', 'check_result_text': 'FAIL: is not found', 'check_result': False},
...
```